### PR TITLE
Info about capturing in unit tests and integration tests

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Info about capturing in unit tests and integration tests

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,6 +136,40 @@ def test_constructor_no_provider():
...
+def test_logger_show_message_flag(mock_load_dotenv, capsys):
+    """Test logger set with show_message flag."""
+    logger = Logger(logger_name="foo", log_level=logging.INFO, show_message=True)
+    logger.logger.debug("Debug message")
+
+    # check captured log output
+    # the log message should not be captured due to log level
+    # but the message should be shown
+    captured_out = capsys.readouterr().out
+    assert "Set LOG_LEVEL environment variable (e.g., INFO, DEBUG)" in captured_out
```
Comment (onmete) [inline CONTRIBUTING.md:168]: Isn't it a bit weird, that you are not searching for the message that you logged at the beginning of the test?

Comment (tisnik) [reply CONTRIBUTING.md:168]: well it won' be there, I will update the example a bit

Comment (onmete) [reply CONTRIBUTING.md:168]: I know it's a copied "real" test. But that test should rather test an empty output, right? Having an at least example clear would be great. :)

Comment (tisnik) [reply CONTRIBUTING.md]: fixed @onmete and TY for a good notice!

**Pull request merged**